### DISCUSSION TITLE
Close old issues

### DISCUSSION THREAD

**Discussion by matsarv** [retrospect-addon/plugin.video.retrospect#1773] [Q&A]
I saw that you cleaned out some old issues, so I looked at the old Swedish issues and came up with this list:

SVT Play
I guess this issue is no longer relevant as SVT changed significantly since 2017:
909

Dplay
Channel don't exist anymore. Following issues are no longer relevant:
1165, 1232, 1432

Viafree
Channel don't exist anymore. Following issues are no longer relevant:
1231, 1270, 1493

Cmore
Channel don't exist anymore. Following issue is no longer relevant:
1525

Discovery+
Channel don't exist in Retrospect anymore. A request to add Discovery+ is in issue 1495:
1615

As usual, it is up to you to decide which issues should be closed.

**Comment by basrieter** [comment]
Thank you very much for the update. I will go and close those then!
